### ISSUE TITLE
New Turing domain: please visit turinglang.org

### ISSUE THREAD

**Issue by fredcallaway** [TuringLang/Turing.jl#2041]
Go here: https://turing.ml/v0.22/tutorials/ and click any of the sidebar links (except linear regression which works) and you'll get routed back to the main page.

**Comment by devmotion** [comment]
This is the old webpage (AFAIK nobody has access to) which also only contains old and outdated docs and tutorials. The Turing docs and tutorials are hosted at turinglang.org nowadays.

**Comment by fredcallaway** [comment]
Oh, that's good to know! I don't suppose there's any way to have turing.ml point to turinglang.org (really, no one has access to that site?) If you google "turing julia" you get turing.ml so I imagine many people will end up there :\

**Comment by itsdfish** [comment]
Agreed. turing.ml is the first result for me and turinglang.org is the last on the first page. Most people are going to land on the old site to find outdated information,  and many will think its an abandoned or poorly maintained project. A redirect would be good if possible.

**Comment by devmotion** [comment]
AFAIK unfortunately really nobody has access to the old turing.ml domain which means it's impossible to add a redirect.

**Comment by cpfiffer** [comment]
Wait, how did we lose the domain?

**Comment by devmotion** [comment]
I have no idea :shrug:

**Comment by cpfiffer** [comment]
I did a little digging -- Freenom [was sued](https://krebsonsecurity.com/2023/03/sued-by-meta-freenom-halts-domain-registrations/) by Meta a little bit ago. They suspended new domain signups and may be difficult to reach for the forseeable future.

Additionally, it looks like the `.ml` TLDs are kind of a mess anyway, since they're the Mali TLD and Mali decided to give them out [for free](https://www.getfreedomain.name/domain/ml/)? 

Perhaps we can find a way to get GitHub to stop serving the files to them. Or are the files cached at the provider somewhere?

**Comment by carolfs** [comment]
Some links are broken on the current website too. For example, in the [Guide page](https://turinglang.org/v0.28/docs/using-turing/guide/), the link to the API is `https://turinglang.org/v0.28/docs/using-turing/guide/%7B%7Bsite.baseurl%7D%7D/docs/library`. It seems like `{{site.baseurl}}` was escaped to `%7B%7Bsite.baseurl%7D%7D`.

**Comment by ParadaCarleton** [comment]
> It was registered with https://www.freenom.com/en/index.html?lang=en. The freenom service stopped functioning a few months ago. Their customer service is not responding to requests. If anyone has a way to claim the domain back, let me know.

Have you tried [contacting Google](https://support.google.com/legal/troubleshooter/1114905?hl=en#ts=9814647%2C1115655%2C13630207%2C12999302%2C9814950%2C1115789%2C1117010%2C1697925) to have it removed? It should be possible to take it down, since it's a copyright violation for Freenom to keep the website up after you asked them to take it down.

**Comment by devmotion** [comment]
The page seems to be served by cloudflare (nameservers audrey.ns.cloudflare.com and wilson.ns.cloudflare.com according to a lookup), maybe the old content is just cached there and there's no intentional copyright violation (the purpose of which I wouldn't understand anyway)?

**Comment by mosullivan93** [comment]
I did a bit of digging as I was curious about this... it seems that Freenom lost management rights to the .ml and .ga ccTLDs around the time this issue was raised. Now, .ml is administered by the Malian government via their ICT agency AGETIC. There are some contact details for the .ml ccTLD found on the [IANA Delegation Record](https://www.iana.org/domains/root/db/ml.html) and the [NIC itself](https://nic.ml/). Not sure if you'd have any lucky if you tried to reach out to them directly, @yebai.

**Comment by penelopeysm** [comment]
I submitted a report to Google here: https://search.google.com/search-console/report-spam

I expect it will be ignored, but maybe if we're very lucky they'll de-list it.

**Comment by shravanngoswamii** [comment]
https://turing.ml is expired and not accessible anymore!✌️

**Comment by shravanngoswamii** [comment]
I don't know how but turing.ml renewed again -- @yebai If you remember setting up any auto-renew/auto-pay for that domain when we had access, is it possible to check if that triggered?

**Comment by penelopeysm** [comment]
I don't think this can be solved. On the bright side, turinglang.org is higher in search results nowadays.
